config/locales/js-en.yml

```diff
@@ -399,16 +399,17 @@ en:
           learn_about: "Learn more about the new features"
           # Include the version to invalidate outdated translations in other locales.
           # Otherwise, e.g. chinese might still have the translations for 10.0 in the 12.0 release.
-          "14_5":
+          "14_6":
             standard:
               new_features_html: >
-                The release contains various new features and improvements, such as: <br>
+                The release contains various new features and improvements, e.g. <br>
```

Comment:
Based on the specification, It should be:

> The release brings various features and improvements for you, e.g.

Reply:
Thank you, that's right, we slightly changed the wording so that it fits the style of the bullet points below.